src/azure-cli-core/tests/test_command_registration.py

```diff
@@ -30,8 +30,8 @@ def tearDownClass(cls):
         logging.shutdown()
 
     @staticmethod
-    def sample_vm_get(resource_group_name, vm_name, opt_param=None, expand=None, custom_headers={},  # pylint: disable=dangerous-default-value, too-many-arguments
-                      raw=False, **operation_config):
+    def sample_vm_get(resource_group_name, vm_name, opt_param=None, expand=None, custom_headers=None, raw=False,
+                      **operation_config):
```

Comment:
I don't think anything ever changed the custom headers, so the dangerous behavior never manifested.